transcom/mymove
# INT B-21114 Address related language changes (Office)

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1010870)

## Summary

The following changes were made on the Office/Prime side in order to make address related language consistent across the MilMove app:

1. "Destination Address" -> "Delivery Address"
2. "Pickup Location" -> "Pickup Address"
3. "Current Address" -> "Pickup Address"
4. "Origin Address" -> "Pickup Address"

### How to test

1. Login as an office user.
2. Access the move detail page of a move, check that the changes above have applied to the shipment cards.
3. Edit a shipment, check for changes.
5. Create a shipment, check for changes.
6. Login as Prime Sim and do the same for all shipment related controls/pages.

## Screenshots
